.zuul.yaml

```diff
@@ -0,0 +1,11 @@
+- project:
+    vars:
+      functest_project_name: "eu-de_zuul_go"
+    check:
+      jobs:
+        - golangci-lint
+        - golang-make-functional
+    gate:
+      jobs:
+        - golangci-lint
```

Comment:
Why duplicate lint job?

Reply:
will not make it worse

Reply:
Don't create useless steps, they take time

Reply:
same as in the other projects: jobs in gate MUST test things (same jobs and even more of those). Otherwise there is no guarantee you are merging working code

Reply:
@gtema Is gate testing something different to what check is testing? Why?

Reply:
1) between check jobs and gate jobs time is passing by and the dependencies are changing (new release of lib x.y is released)
2) check job can run tests "assuming" dependent changes are merged (helps checking changes in projects with dependencies). Gate job will then "hold" merge until the dependent changes get really merged
3) see https://zuul-ci.org/docs/zuul/discussion/gating.html#project-gating for detailed explanation

Reply:
Nice